src/xunit.performance.analysis/Program.cs

```diff
@@ -454,6 +310,23 @@ private static IEnumerable<TestIterationResult> ParseOneXmlFile(string path)
                 var runId = perfElem.Attribute("runid").Value;
                 var etlPath = perfElem.Attribute("etl").Value;
 
+                foreach(var metrics in perfElem.Descendants( "metrics" ))
+                {
+                    foreach(var metric in metrics.Elements())
+                    {
+                        var metricName = metric.Name.LocalName;
+                        var unit       = metric.Attribute("unit").Value;
+
+                        //
+                        // Populate the set of all collected metrics 
+                        //
+                        if(Properties.AllMetrics.ContainsKey( metricName ) == false)
```

Comment:
Yes, this is just based on the existing conventions in the code.  There isn't a tool that we are using to enforce this - same goes for other repos like coreclr and corefx.